recipe/meta.yaml

```
@@ -11,22 +11,24 @@ source:
   sha256: "{{ sha256 }}"
 
 build:
-  noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  skip: True # [py<37]
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv"
 
 requirements:
   host:
-    - pip
     - python
     - pytest-runner >=2.7
     - setuptools_scm >=3.1
+    - wheel
```

```suggestion
    - setuptools
    - wheel
```
Missing `setuptools` as a Python build backend, see https://github.com/inveniosoftware/dictdiffer/blob/v0.9.0/setup.py#L17. We explicitly add it to `host` in such cases.